opentelemetry-api/src/opentelemetry/context/contextvars_context.py

```diff
@@ -12,16 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from contextvars import ContextVar
-from sys import version_info
 
 from opentelemetry.context.context import Context, _RuntimeContext
 
-if (3, 5, 3) <= version_info < (3, 7):
-    import aiocontextvars  # type: ignore # pylint:disable=unused-import,import-error
```

Comment:
is this import not still required?

Reply:
Ah, I see, yes, this seems to be required still.